spec/features/visitors/sign_up_spec.rb

```diff
@@ -15,7 +15,7 @@
   #   Then I see a message that I need to confirm my email address
   scenario 'visitor can sign up with valid email address' do
     sign_up_with('test@example.com')
-    expect(page).to have_content(
+    expect(page.body).to include(
```

Comment:
Hmm I see..  the use of raw() in the slim and <strong> in the string data makes me feel funny.  Can we avoid it?  Should I just get over it?